CHANGELOG.adoc

```diff
@@ -122,6 +122,10 @@ This incorporates the following executed proposals:
 
 Added the ic-btc-adapter binary to the cache.
 
+=== chore: updates starter project for Node 18
```

Comment:
Would you please move this up, under the `== dfx` section, rather than here under the `== replica` section?